src/app/components/client/ExposureCardDataClass.tsx

```diff
@@ -0,0 +1,98 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { ReactElement } from "react";
+import styles from "./ExposureCard.module.scss";
```

Comment:
Not sure how doable it is to extract them now, but the idea behind CSS modules is that one can safely modify those styles without unintentionally affecting styles elsewhere, so ideally, each component has its own CSS module containing only the styles relevant to that module.